src/libnmea_navsat_driver/parser.py

```diff
@@ -277,7 +277,7 @@ def parse_nmea_sentence(nmea_sentence):
         False if the sentence could not be parsed.
     """
     # Check for a valid nmea sentence
-
+    nmea_sentence = nmea_sentence.strip("\r\n") # Cut possible carriage return or new line of NMEA Sentence
```

Comment:
Please add an extra space before the comment to comply with [PEP8 for inline comments](https://www.python.org/dev/peps/pep-0008/#inline-comments). Alternatively, it might be better to remove the comment, since it is simply restating what the code is doing.

```suggestion
    nmea_sentence = nmea_sentence.strip("\r\n")  # Cut possible carriage return or new line of NMEA Sentence
```

Reply:
Actually, after looking at issue #89, I have another request. Can you change this to strip all whitespace, so that it's consistent with the checksum function in this package?

```suggestion
    nmea_sentence = nmea_sentence.strip()
```

Reply:
Fixed it.